src/reproschema_to_fhir/fhir.py

```diff
@@ -8,48 +8,39 @@
 from fhir.resources.questionnaire import Questionnaire
 from fhir.resources.valueset import ValueSet
 from fhir.resources.codesystem import CodeSystem
+from datetime import datetime, timezone
 
 from .config import Config
 
-# TODO: migrate these globals to the config
-CODESYSTEM_URI = os.getenv('CODESYSTEM_URI')
-VALUESET_URI = os.getenv('VALUESET_URI')
-QUESTIONNAIRE_URI = os.getenv('QUESTIONNAIRE_URI')
 
-def generate_code_system(
-    options_json, id_str: str
-) -> dict:
+def generate_code_system(options_json, id_str: str, config) -> dict:
     """
     Helper function to generate a FHIR CodeSystem resource from a reproschema options json.
     """
     # default headers for codesystem
     codeSystem = dict()
-    # id = questionnaire + linkId
-    # id = id.replace("_", "-")
-    # id = id.lower()
-
     codeSystem[f"resourceType"] = f"CodeSystem"
     codeSystem[f"id"] = id_str
     codeSystem[f"text"] = {
         f"status": f"generated",
         f"div": f'<div xmlns="http://www.w3.org/1999/xhtml">Placeholder</div>',
     }
 
-    codeSystem[f"url"] = f"{CODESYSTEM_URI}{id_str}"
+    codeSystem[f"url"] = f"{config.get_codesystem()}{id_str}"
     codeSystem[f"version"] = f"1.4.0"
     codeSystem[f"name"] = id_str.capitalize().replace("_", "")
     codeSystem[f"title"] = id_str
     codeSystem[f"status"] = f"active"
-    codeSystem[f"date"] = f"2023-11-20T11:33:15-05:00"
+    codeSystem[f"date"] = str(datetime.now(timezone.utc))
```

Comment:
you can use `strftime`, which will be much more explicit than this. it's hard to tell what the date format created here is, but with a format code and strftime it's obvious: https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior